ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -19,6 +24,6 @@ wait "$!"
 ret="$?"
 set -e
 
-cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}"
+sed 's/password: .*/password: REDACTED/' "/tmp/installer/.openshift_install.log" >"${ARTIFACT_DIR}/.openshift_install.log"
```

Comment:
why is the openshift install log here have the password in the first place?

Reply:
I was making it consistent with elsewhere, but you are right. Removed this change from the PR.